src/jitlayers.cpp

```diff
@@ -134,8 +134,10 @@ void addOptimizationPasses(legacy::PassManagerBase *PM, int opt_level,
             PM->add(createLowerPTLSPass(dump_native));
         }
         PM->add(createLowerSimdLoopPass());        // Annotate loop marked with "loopinfo" as LLVM parallel loop
+#if !defined(_OS_EMSCRIPTEN_)
```

Comment:
The pass manager should be platform independent (though the pass itself may branch on a target triple). Is multiversioning not sufficiently disabled when we pass in a singleton target (the default)?

Reply:
I think it's fine to drop from this patch. Since we dump the bc on the wasm bootstrap path, I don't think we ever reach here anyway. We can revisit if we get native bootstrapping working for wasm.